lms/djangoapps/discussion/rest_api/serializers.py

```diff
@@ -325,6 +339,13 @@ def get_unread_comment_count(self, obj):
             return obj["unread_comments_count"] + 1
         return obj["unread_comments_count"]
 
+    def get_preview_body(self, obj):
+        """
+        Returns a cleaned and truncated version of the thread's body to display in a
+        preview capacity.
+        """
+        return Truncator(strip_tags(self.get_rendered_body(obj))).words(10, )
```

Comment:
@xitij2000 You've mentioned that you have doubts about this. Is the final decision to keep the truncation feature in this PR?